workitem/workitem_repository.go

```diff
@@ -102,6 +117,30 @@ func (r *GormWorkItemRepository) LoadByID(ctx context.Context, id uuid.UUID) (*W
 	return ConvertWorkItemStorageToModel(wiType, res)
 }
 
+// LoadBatchByID returns work items for the given ids
+func (r *GormWorkItemRepository) LoadBatchByID(ctx context.Context, ids []uuid.UUID) ([]*WorkItem, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "workitem", "loadBatchById"}, time.Now())
+	res, err := r.LoadBatchFromDB(ctx, ids)
+	if err != nil {
+		return nil, errs.WithStack(err)
+	}
+	workitems := []*WorkItem{}
+	for _, ele := range res {
+		wiType, err := r.witr.LoadTypeFromDB(ctx, ele.Type)
+		if err != nil {
+			return nil, errors.NewInternalError(ctx, err)
```

Comment:
Maybe it is worthwhile to log the error here as well?

Reply:
done in https://github.com/fabric8-services/fabric8-wit/pull/1592/commits/a3e514baf7e0a925f5ec5d76b57886d04251f2a6